lib/spack/spack/test/link_tree.py

```diff
@@ -38,6 +38,8 @@ class LinkTreeTest(unittest.TestCase):
 
     def setUp(self):
         self.stage = Stage('link-tree-test')
+        # FIXME : possibly this test needs to be refactored to avoid the explicit call to __enter__ and __exit__
```

Comment:
@tgamblin I really like the idea, and besides `Stage.destroy` is already there. I guess we just need a `Stage.create` for simmetry...